shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/format/impl/MySQLFormatSQLVisitor.java

```diff
@@ -68,24 +69,41 @@
 import org.apache.shardingsphere.sql.parser.autogen.MySQLStatementParser.WhereClauseContext;
 import org.apache.shardingsphere.sql.parser.autogen.MySQLStatementParser.WithClauseContext;
 
+import java.util.Properties;
+
 /**
  * MySQL Format SQL visitor for MySQL.
  */
+@NoArgsConstructor
 @Getter
 @Setter
 public abstract class MySQLFormatSQLVisitor extends MySQLStatementBaseVisitor<String> {
 
     private StringBuilder result = new StringBuilder();
 
-    private boolean uperCase = true;
+    private boolean upperCase = true;
 
     private boolean parameterized = true;
 
     private int indentCount;
 
     private int lines;
 
-    private final int projectionsCountOfLine = 3;
+    private int projectionsCountOfLine = 3;
+
+    MySQLFormatSQLVisitor(final Properties props) {
+        if (null != props) {
```

Comment:
Is it possible to remove this `if (null != props)`?

Reply:
No.